src/orderbook.ts

```diff
@@ -47,6 +49,10 @@ export class Orderbook {
     this.bids = new Map();
   }
 
+  pair() {
+    return `${this.baseToken}/${this.quoteToken}`;
```

Comment:
I'm surprised why typescript is not enforcing you to define the return type for functions if you have `strict: true`. I think probably there's something wrong with the configuration. I would encourage you to force that, since for us it have help prevent many mistakes. Also it's easier to understand the signature.

Reply:
I assume, typescript can auto-derive the return types in most cases. The invoking this method somewhere else the code completion tells you the return type, so I think it's fine?